### DISCUSSION TITLE
Create a private app without user entry on the database.

### DISCUSSION THREAD

**Discussion by Ronak-maker** [osiset/laravel-shopify#680] [Q&A]
I want to create a private app, on the installation time don't want to any store user entry on the database user table. it is possible? and what steps I have to follow for this?

**Comment by gnikyt** [comment]
Not without heavily modifying the package no. By default it relies on a users table.

**Reply by Ronak-maker** [reply]
Hey, thank you for the reply. 
Do I have to do different settings for a private app? If yes, what are the settings to do that?

**Reply by gnikyt** [reply]
Just make a "Custom App" in partner dashboard, it can be only installed on one store then.
